sdk/src/client/node_api/indexer/query_parameters.rs

```diff
@@ -9,41 +9,35 @@ use crate::types::block::{address::Bech32Address, output::TokenId, slot::SlotInd
 
 // https://github.com/iotaledger/inx-indexer/tree/develop/pkg/indexer
 
-pub trait QueryParameter {
+pub trait QueryParameter: Serialize + Send + std::marker::Sync {
```

Comment:
shouldn't need the Sync path?

Reply:
Yeah not sure, maybe a clippy bug, but without it I get this
```
error: future cannot be sent between threads safely
  --> sdk/src/client/node_api/indexer/mod.rs:24:10
   |
24 |     ) -> Result<OutputIdsResponse> {
   |          ^^^^^^^^^^^^^^^^^^^^^^^^^ future returned by `get_output_ids` is not `Send`
   |
note: future is not `Send` as this value is used across an await
  --> sdk/src/client/node_api/indexer/mod.rs:45:18
   |
41 |                     query_parameters.to_query_string().as_deref(),
   |                     ---------------- has type `&impl QueryParameter` which is not `Send`
...
45 |                 .await?;
   |                  ^^^^^ - `query_parameters` is later dropped here
   |                  |
   |                  await occurs here, with `query_parameters` maybe used later
help: consider moving this into a `let` binding to create a shorter lived borrow
  --> sdk/src/client/node_api/indexer/mod.rs:41:21
   |
41 |                     query_parameters.to_query_string().as_deref(),
   |                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   = note: `impl QueryParameter` doesn't implement `std::marker::Sync`
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#future_not_send
```

Reply:
No, I mean it should be just ```Send + Sync``` without the path

Reply:
Ah, changed